app/config/services/repositories.xml

```diff
@@ -202,5 +202,7 @@
             <factory service="doctrine" method="getRepository" />
             <argument>AppBundle\Entity\ReferentTag</argument>
         </service>
+
+        <service id="AppBundle\Repository\AdherentEmailSubscriptionHistoryRepository" />
```

Comment:
maybe `EmailSubscriptionHistory(Repository|Type|Command|Manager)` (without `Adherent`) will it be enough?

Reply:
Yes ;-)